app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new
+    redirect_to home_path if current_user
+  end
+
+  def create
+    user = User.where(email: params[:email]).first
```

Comment:
I would recommend using `find_by` instead of `where(...).first`. In Rails 4, it is more efficient method for single record search.